packages/cycle-scripts/configs/webpack.config.js

```diff
@@ -1,4 +1,4 @@
-'use strict'
+module.exports = language => `'use strict'
```

Comment:
If I get this right, this is not a configuration file for webpack anymore, but a precompiled template that will compile to a webpack configuration factory? If so filename is perhaps misleading, no?